### PR TITLE
modify DataServlet to display alternative message on /data

### PR DESCRIPTION
**Pull request by maricarol3355**
Part one of the Tutorial (Servlets)
Site: https://maricarol-step-2020.uc.r.appspot.com
Trying https://maricarol-step-2020.uc.r.appspot.com/data should display "Hello Marissa!"

### CODE REVIEW

**Comment by maricarol3355** [comment]
Part two of the Tutorial (Servlets)
Site: https://maricarol-step-2020.uc.r.appspot.com
Add button "fetch a hello" to display message at the bottom of the page

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -21,12 +21,12 @@
...
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data")
+@WebServlet("/hello")
 public class DataServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    response.getWriter().println("<h1>Hello Marissa!</h1>");
```
Comment (ptai7) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:30]: Clicking the button "Fetch a Hello!" button currently returns "\<h1>Hello Marissa!\</h1>".

**Review by ptai7 — APPROVED** [review]
Looks good to me, just left a comment about the Hello Marissa display.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (maricarol3355) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:30]: I'm going leave it as a TODO and just remove the tags. I spent too much time trying to resolve the issue, so I'll come back to this later.

**Pull request merged**